src/index.js

```diff
@@ -26,15 +27,16 @@ const pageLoadDispatcher = new PageLoadDispatcher();
 const ajaxDispatcher = new AjaxDispatcher();
 const elementTimingDispatcher = new ElementTimingDispatcher();
 const webVitalsDispatcher = new WebVitalsDispatcher();
+const memoryUsageDispatcher = new MemoryUsageDispatcher();
 
 visibilityObserver.addListener(pageLoadDispatcher);
 visibilityObserver.addListener(ajaxDispatcher);
 visibilityObserver.addListener(elementTimingDispatcher);
-
 // DON'T: visibilityObserver.addListener(webVitalsDispatcher);
 // we're not adding web vitals dispatcher to the visibility observer because we
 // don't control the implementation of how metric are collected and I believe
-// that web-vitals already does some visibility change handling
+// that web-vitals already does some visibility change handling.
+// We don't want to add memoryUsageDispatcher to visibility obeserver as well.
```

Comment:
Why not? :)

Reply:
IMHO it doesn't matter if the page is visible or not if it still uses memory :)

Reply:
True, but I'm not sure how reliable the measurements will be. Let's leave it as-is and we'll do some tests on real world data.

Reply:
I like the idea, let's see how it will work once we integrate that and adjust if needed 👍